### DISCUSSION TITLE
Adding custom styles on an existing element clears them

### DISCUSSION THREAD

**Discussion by l-mtc** [awcodes/filament-tiptap-editor#222] [Q&A]
e.g` <h2><span style="color: rgb(241, 111, 44); font-weight:900;"> when I add the font weight and save it clears the code`

**Comment by awcodes** [comment]
Are you adding them through the source code editor? Prosemirror is really strict about what it allows unless it's specifically told to allow it. It's possible there's a bug in the php parser, too. But I need a little more insight into what you are actually doing.

**Comment by l-mtc** [comment]
Yea exactly, adding it to the source code editor, I'm literally just adding additional styles, so I added a colour for a part of an h2 and wanted to add a font weight and when I save the font weight disappears, I think it's the semicolon because that didn't get added after the colour by default.

**Comment by awcodes** [comment]
I think the issue is that font-weight is getting processed by the Bold extension. 

I've tested and font-weight: 900; is getting converted to strong tags.

Then the color is working fine for me.

![Screenshot 2023-11-29 at 3 09 23 PM](https://github.com/awcodes/filament-tiptap-editor/assets/3596800/5d7908a3-fab9-475d-936d-fcad87dd35f5)
![Screenshot 2023-11-29 at 3 10 09 PM](https://github.com/awcodes/filament-tiptap-editor/assets/3596800/756c010b-895d-4ae7-a0aa-201874fc762c)

**Comment by l-mtc** [comment]
@awcodes it seems like source code doesn't save, I tried adding style="display: flex;
    align-items: center;
    gap: 10px;" and it just clears the code all together - got any ideas or workarounds?
